tiledb/sm/misc/logger.h

```diff
@@ -184,6 +184,12 @@ inline Status LOG_STATUS(const Status& st) {
 }
 #endif
 
+/** Logs an error and exits with a non-zero status. */
+inline void LOG_FATAL(const std::string& msg) {
+  global_logger().error(msg.c_str());
```

Comment:
What does the `global_logger()` method return if TileDB is built in non-verbose mode?

Reply:
`global_logger()` is unaffected by the verbose flag. This unconditionally log the error message, even if verbose mode is disabled. That seems appropriate for a fatal error message.